code/style.css

```diff
@@ -1,5 +1,223 @@
-/* If you can see a peach background, the css file is correctly linked. Feel free to change or remove this styling! */
+@import url('https://fonts.googleapis.com/css2?family=Raleway:wght@300;400&display=swap');
 
 body {
-  background-color: peachpuff;
+  background-color: rosybrown;
+  font-family: 'Raleway', sans-serif;
+  color: whitesmoke;
+  margin: 0 60px;
 }
+
+header {
+  display: flex;
+  background-image: url("./img/background-peony.jpg");
+  background-size: cover;
+  margin: 20px 0;
+  border: 2px solid whitesmoke;
+  padding: 15px;
+}
+
+.heading-desktop {
+  display: none;
+}
+
+.heading-mobile h1 {
+  margin: 0;
+  padding-left: 10px;
+  font-size: 32px;
+}
+
+menu{
+  display: none;
+  align-items: center;
+  justify-content: center;
+  padding-inline-start: 0;
+}
+
+.hamburger {
+  display: flex;
+  background-color: rgb(180, 141, 141);
+  height: 30px;
+  width: 30px;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.hamburger div {
+  width: 30px;
+  height: 5px;
+  background: #fff;
+}
+
+menu a {
+  display: inline-flex;
+  padding: 10px;
+  margin: 2px;
+  background: white;
+  color: lightcoral;
+  font-weight: bold;
+  text-decoration: none;
+  border-radius: 3px;
+  flex-basis: 20%;
+  justify-content: center;
+}
+
+menu a:hover{
+  background: lightcoral;
+  color: white;
+}
+
+h1 {
+  text-align: left;
+  font-size: 50px;
+  margin: 20px;
+  font-family: 'Raleway', sans-serif;
+  text-shadow: 1px 1px gray;
+}
+
+h2 {
+  font-family:'Raleway', sans-serif;
+  padding-left: 30px;
+}
+
+main {
+  border: 2px dotted lightcoral;
+  padding: 10px;
+  margin-bottom: 20px;
+}
+
+.highlighter-block {
+  background-color: #c8a3a3;
+  display: inline-block;
+  padding: 10px;
+  border-radius: 20px;
+  margin: 5px;
+  box-shadow: 0px 0px 10px 1px whitesmoke;
+}
+
+form {
+  background-color: #c8a3a3;
+  display: block;
+  padding: 20px;
+  margin: 15px;
+  border-radius: 20px;
+  box-shadow: 0px 0px 10px 1px whitesmoke;
+}
+
+.video {
+  width: 560px;
+  height: 315px;
+}
+
+.note {
+  font-style: italic;
+}
+
+a {
+  color: inherit;
+}
+
+a:hover {
+  background: #d7bcbc
+}
+
+.video {
+  display: block;
+  margin: 60px;
+}
+
+.product-grid{
+  display: flex;
+  flex-direction: column;
+}
+
+.product-card{
+  width: 90%;
+  margin: 5px;
+  padding: 5%;
+  border: 1px solid whitesmoke;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  background-color: #ecc0c0cc;
+}
+
+.product-image{
+  height: 120px;
+  width: 100px;
+  border: 1px solid gray;
+}
+
+.product-content {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.product-name{
+  font-size: larger;
+  margin-bottom: 5px;
+}
+
+.product-description{
+  font-size: 12px;
+  text-align: center;
+}
+
+.product-link {
+  font-size: smaller;
+}
+
+@media (min-width: 992px) {
```

Comment:
One idea is to have more than one media query, in that case, you could decide on another layout for tablet size! :)